examples/use_object_storage.md

```diff
@@ -35,6 +35,44 @@ upctl object-storage user create ${prefix}service --username ${prefix}user
 upctl object-storage access-key create ${prefix}service --username ${prefix}user
 ```
 
+Save the access key ID and secret access key from the output - you'll need these for S3 access.
+
+Attach a policy to grant the user access to buckets:
+
+1. **Using the UpCloud API:**
+
+   First, get your service UUID:
+   ```sh
+   upctl object-storage list
+   ```
+
+   Then attach the policy (replace `<service-uuid>` with your UUID):
+   ```sh
+   curl -X POST "https://api.upcloud.com/1.3/object-storage-2/<service-uuid>/users/${prefix}user/policies" \
+     -H "Authorization: Bearer ${UPCLOUD_TOKEN}" \
+     -H "Content-Type: application/json" \
+     -d '{"name": "ECSS3FullAccess"}'
+   ```
+
+   A successful response returns HTTP status 204.
+
+2. **Using the UpCloud Control Panel:**
+   Navigate to Object Storage → Users → Select user → Attach Policy → ECSS3FullAccess
```

Comment:
Instead of these, we would want to have example on how to do this `upctl`. Problem is that we do not currently have commands to do that: #647

Reply:
@kangasta I agree having object storage managed by `upctl` is desirable. But I would leave current example as-is, until the new commands are implemented. This ensures users and tests reflect the current state, instead of future plans.

After we merge this, I would happily propose another PR for #647 and additional object storage commands to allow flexible management of object storage.